### PR TITLE
Introduce a global temperature property

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Review by jmartisk — CHANGES_REQUESTED** [review]
The changes in `samples/review-triage/pom.xml` look accidental :)

**Comment by geoand** [comment]
Good catch!

File: core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/config/LangChain4jConfig.java [hunk 1]
```diff
@@ -31,6 +32,11 @@ public interface LangChain4jConfig {
     @ConfigDocDefault("10s")
     Optional<Duration> timeout();
 
+    /**
+     * Global temperature for LLM APIs
```
Comment (jmartisk) [inline core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/config/LangChain4jConfig.java:36]: Should we say what the allowed range is?

(And here we are, as I expressed my concern about creating similar abstractions - are we sure that all models and model providers will have the same allowed range for temperature?)

Comment (jmartisk) [reply core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/config/LangChain4jConfig.java:36]: Then it's questionable whether to have this abstraction at all, but ok, a different number range isn't that hardcore, I could imagine worse :)

**Review by jmartisk — APPROVED** [review]

**Pull request merged**